great-idea-website/index.html

```diff
@@ -14,50 +15,78 @@
-
-  Vision
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  Contact
-  123 Way 456 Street
-  Somewhere, USA
-  1 (888) 888-8888
-  sales@greatidea.io
-
-  Copyright Great Idea! 2018
-  
-  </body>
-</html>
\ No newline at end of file
+<body>
+  <nav>
+    <div class="menu">
+      <a href='#'>Services</a>
+      <a href='#'>Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+    </div>
+    <div class="menu-logo">
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </div>
+  </nav>
+  <header>
+    <div class="intro">
+      <h2>Innovation</h2>
+      <h2>On</h2>
+      <h2>Demand</h2>
+      <button>Get Started</button>
+    </div>
+    <img src="img/header-img.png" alt="Image of a code snippet.">
+
+  </header>
+  <hr>
+  <section>
+    <div class="box-one">
+      <h4>Features</h4>
+      <p> Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    <div class="box-one">
+      <h4>About</h4>
+      <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+        Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+    </div>
+    <div class='middle-img'>
+      <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    </div>
+    <div class="section-two">
+      <div class="box-two">
+        <h4>Services</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+      <div class="box-two">
+        <h4>Product</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+
+      <div class="box-two">
+        <h4>Vision</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+    </div>
+  </section>
+
+  <hr>
+
+  <section class="address">
+    <h4>Contact</h4>
+    <address>
+      <p>123 Way 456 Street <br>Somewhere, USA</p>
+      <p>1 (888) 888-8888</p>
+      <p><a href='#'>sales@greatidea.io</a></p>
```

Comment:
Transform the mail paragraph  into an anchor is a great touch. You could finish in great style adding `mailto:sales@greatidea.io` to `href` value. 
Have a look here: https://css-tricks.com/snippets/html/mailto-links/